src/ussd_pmtct.js

```diff
@@ -71,21 +77,80 @@ go.app = function() {
                 }
             });
 
-            return http.get(subscription_base_url + endpoint, {
-                    params: {
-                        "to_addr": msisdn
+            return http
+            .get(subscription_base_url + endpoint, {
+                params: {
+                    "to_addr": msisdn
```

Comment:
@gsvr should this not be ```"query": "to_addr=" + to_addr``` (asking because saw that getVumiContactByMsisdn function was changed to be in this format)